README.md

```diff
@@ -27,8 +27,9 @@ vpc:
         - sg-123456789
         ...
 ```
-For each lambda function
+For each lambda function.
       
+> Note: The plugin can be used with the core serverless vpc config. Take a look at [official documentation](https://www.serverless.com/framework/docs/providers/aws/guide/functions#vpc-configuration). 
```

Comment:
Maybe this note should be combined with https://github.com/amplify-education/serverless-vpc-discovery/pull/36/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R101